src/components/Trading/TradingDialog.tsx

```diff
@@ -24,10 +26,26 @@ interface TradingDialogProps {
   sendTransaction: (transaction: Transaction) => void
 }
 
+function getAssetFromPath(pathname: string) {
+  if (matchesRoute(pathname, routes.tradeAsset("*", undefined, "*"))) {
```

Comment:
[Super minor] Only change this if you need to touch things here once more anyway:

That kind of code is usually better readable / comes with less cognitive overhead if you turn the `if X then some block of Y` into `if not X exit early; some block of Y` 🙂

Reply:
I decided not to change it because in this case we'd need two `return undefined` statements since the function returns `undefined` when either the first or the second `if`-statement is not true.

But you're probably right that it's more intuitive to exit early and I will try to keep it in mind and use it in future scenarios :)

Reply:
Fair enough 😉